tabs/mission_control.js

```diff
@@ -2809,7 +2815,9 @@ TABS.mission_control.initialize = function (callback) {
     function checkAltElevSanity(resetAltitude, checkAltitude, elevation, P3Datum) {
```

Comment:
Better to change `P3Datum `to `P3Value `since it no longer refers to a datum any more unlike previously ?

Reply:
I've renamed the variable.